libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -528,12 +547,24 @@ bool AP_Arming::board_voltage_checks(bool report)
 #if HAL_HAVE_BOARD_VOLTAGE
     // check board voltage
     if ((checks_to_perform & ARMING_CHECK_ALL) || (checks_to_perform & ARMING_CHECK_VOLTAGE)) {
-        if(((hal.analogin->board_voltage() < AP_ARMING_BOARD_VOLTAGE_MIN) || (hal.analogin->board_voltage() > AP_ARMING_BOARD_VOLTAGE_MAX))) {
-            check_failed(ARMING_CHECK_VOLTAGE, report, "Check board voltage");
+        const float bus_voltage =  hal.analogin->board_voltage();
+        if(((bus_voltage < _vbus_min) || (hal.analogin->board_voltage() > AP_ARMING_BOARD_VOLTAGE_MAX))) {
```

Comment:
Well, you *did* go to the trouble of pulling out a temporary variable.  Suggest reusing it here :-)